### PR TITLE
Update get-fundraiser.js

### PR DESCRIPTION
**Pull request by AntoineMarnat**
Update as comments for fundraisers.

### CODE REVIEW

**Comment by AntoineMarnat** [comment]
@cbeauchesne , should I add all the children waypoints of Presles for VTNO, or the main waypoint is enough ?

File: src/js/get-fundraiser.js [hunk 1]
```diff
@@ -1,20 +1,102 @@
 const mapping = {
-  // 14274: { // France
+  // France
+  // 14274: { // FFME - Rockclimber
   //   url: ' https://www.helloasso.com/associations/rockclimber/formulaires/1/widget'
   // },
+  
+  //Greenspit
```
Comment (brunobesson) [inline src/js/get-fundraiser.js:7]: please add space after `//`, it eases the reading

**Review by brunobesson — APPROVED** [review]
If too many cases are added, then this solution won't last :)

File: src/js/get-fundraiser.js (hunk 1 above)
Comment (AntoineMarnat) [reply src/js/get-fundraiser.js:7]: OK, I'll do.

We should find a way to transmit to the children ...

**Comment by cbeauchesne** [comment]
@brunobesson 

> If too many cases are added, then this solution won't last :)

Definitly :)

**Pull request merged**